microsoft-edge/devtools-guide-chromium/javascript/snippets.md

```diff
@@ -22,149 +22,158 @@ ms.date: 05/04/2021
+
+**Note:** The security settings of most webpages block from loading other scripts in snippets. For this reason, you must include all your code in one file.
 
 
 <!-- ====================================================================== -->
 ## Open the Snippets tab
 
-The **Snippets** tab, in the **Navigator** pane on the left, lists your Snippets.  When you want to edit a Snippet, you need to open it from the **Snippets** tab:
+The **Snippets** tab is located in the **Navigator** pane, on the left of the **Sources** tool.
 
-![The Snippets tab](./snippets-images/javascript-sources-snippets-pane.png)
+To open the **Snippets** tab:
 
-### Open the Snippets tab with a mouse
+1. To open DevTools, right-click the webpage, and then select **Inspect**. Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS). DevTools opens.
 
-1. Select the **Sources** tab.  The **Sources** tool appears:
+1. In DevTools, on the Activity Bar, select the **Sources** tab. If that tab isn't visible, click the **More tools** (![More tools icon](./snippets-images/more-tools-icon.png)) button.
 
-   ![The Sources tool with the Page tab open on the left](./snippets-images/javascript-sources-page-pane.png)
+   ![The Sources tool with the Page tab open on the left](./snippets-images/sources-tool-page-pane.png)
 
 1. In the **Navigator** pane (on the left), select the **Snippets** tab.  To access the **Snippets** option, you might need to click the **More tabs** (![More tabs](./snippets-images/more-tabs-icon.png)) button.
 
-### Open the Snippets tab with the Command Menu
+#### Open the Snippets tab with the Command Menu
+
+You can also open the **Snippets** tab with the Command Menu:
 
 1. Select anything in DevTools, so that DevTools has focus.
 
 1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the Command Menu.
 
-1. Type **Snippets**, select **Show Snippets**, and then press **Enter** to run the command:
+1. Start typing "Snippets", select **Show Snippets**, and then press **Enter** to run the command:
 
-   ![The Show Snippets command](./snippets-images/javascript-search-show-snippets.png)
+   ![The Show Snippets command](./snippets-images/command-menu.png)
 
 
 <!-- ====================================================================== -->
-## Create Snippets
+## Create new snippets
 
-### Create a Snippet through the Sources tool
+To create a new snippet from the **Snippets** tab:
 
 1. [Open the Snippets tab](#open-the-snippets-tab).
 
 1. Click **New snippet**.
 
-1. Enter a name for your Snippet, and then press **Enter**:
+1. Enter a name for your snippet, and then press **Enter**:
 
-   ![Name a Snippet](./snippets-images/javascript-sources-snippets-naming.png)
+   ![A new, empty, snippet in the Sources tool](./snippets-images/new-snippet.png)
 
-### Create a Snippet through the Command Menu
+#### Create a new snippet with the Command Menu
```

Comment:
```suggestion
#### Create a new snippet from the Command Menu
```
"with" and "Note" are overused, like wildcards.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with